### DISCUSSION TITLE
Fresh install, kube-proxy is missing.

### DISCUSSION THREAD

**Discussion by pa-tempus** [k3s-io/k3s#6466] [Q&A]
**Environmental Info:**
k3s version v1.25.3+k3s1 (f2585c16)
go version go1.19.2

Node(s) CPU architecture, OS, and Version: 
v1.25.3+k3s1 (f2585c16)

Cluster Configuration:

**Describe the bug:**
kube-proxy is missing

**Steps To Reproduce:**
Fresh install, its just missing completely.  Tested it twice.

**Expected behavior:**
It should be there by default unless I disabled it via install argument.

**Comment by brandond** [comment]
Can you define what you mean by "missing"? How did you make this determination? K3s doesn't run any of the core Kubernetes components as pods or separate processes. All you will ever see processes for k3s and containerd, and then of course your pods.

https://github.com/k3s-io/k3s#how-is-this-lightweight-or-smaller-than-upstream-kubernetes
> The memory footprint is reduced primarily by running many components inside of a single process. This eliminates significant overhead that would otherwise be duplicated for each component.

**Comment by pa-tempus** [comment]
I guess I was confused.  I ran minikube in my lab and made an edit to kube-proxy for metallb..but noticed it was missing when I started using k3s.
